ui/app/templates/components/secret-list-header.hbs

```diff
@@ -16,11 +16,9 @@
       <h1 class="title is-3">
         <Icon @glyph={{or @model.engineType "secrets"}} @size="xl" class="has-text-grey-light" />
         {{@model.id}}
-        {{#if (eq @model.options.version 2)}}
```

Comment:
Only KV type backends should have this badge. This component is used in every type of backend, so we still need some check here to make sure we're only rendering on mount type KV

Reply:
Thanks for pointing that out @hashishaw. I'll update and add a test. Quick question regarding `engineType`. I see in some places there is check for both `kv` and `generic` types. Is `generic` a pre kv type from older versions that we are now mapping to kv?

Reply:
That is a great question. Let me check and get back to you 👀

Reply:
Yes, `generic` was "renamed" to `kv` but we keep it for backwards compatibility 👍